lumicks/pylake/piezo_tracking/tests/conftest.py

```diff
@@ -95,8 +95,10 @@ def piezo_tracking_test_data(poly_baseline_data, camera_calibration_data):
 
         def implicit_trap_position_equation(x):
             trap_trap_dist = x - trap2_ref
-            displacement = 2 * (force + baseline(x)) / stiffness_um
-            return (tether_dist + 2 * bead_radius + displacement - trap_trap_dist) ** 2
+            displacement = 2 * (force + baseline(x)) / stiffness_um  # noqa: B023
```

Comment:
what's a noqa?

Reply:
It silences the linter about B023.

In this case, the linter warns us about the fact that that function doesn't bind `tether_dist` and `force` from the enclosing loop. That might be an issue if you call the function _after_ the loop, but in this case the behavior you get is fine since the function is only ever used inside the loop.